config/gatekeeper/policy_v1beta1_podsecuritypolicy_gatekeeper-admin.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
I'm wondering if this will have any impact on the OpenShift installation. cc @willkutler

Reply:
I have added an update permission to policies which was an error I was seeing while deploying on Openshift.